pkg/chain/ethereum/connect.go

```diff
@@ -6,6 +6,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/keep-network/keep-core/pkg/beacon/relay/config"
```

Comment:
I looked at all the files with this import and all of them but `connect.go` reference `"github.com/keep-network/keep-core/pkg/beacon/relay/chain"` as well. I wonder if it makes sense to have a `config` package with `chain.go` inside. Perhaps we should rename `Chain` to `Config` and move it to `chain.go` in `pkg/beacon/relay/chain` and remove `pkg/beacon/relay/chain` altogether. What do you think?